### PR TITLE
Add new method to detect if root request is an API request

### PR DESCRIPTION
**Pull request by tsteur**
See changes.

### CODE REVIEW

File: plugins/PrivacyManager/templates/gdprOverview.twig [hunk 1]
```diff
@@ -14,6 +14,8 @@
         </p>
     </div>
 
+    {{ postEvent('Template.afterGDPROverviewIntro') }}
```
Comment (sgiehl) [inline plugins/PrivacyManager/templates/gdprOverview.twig]: Is this supposed to be in this PR? Seems unrelated

**Review by sgiehl — APPROVED** [review]
Besides the comment, LGTM

File: plugins/PrivacyManager/templates/gdprOverview.twig (hunk 1 above)
Comment (tsteur) [reply plugins/PrivacyManager/templates/gdprOverview.twig]: Yep it was supposed to be, forgot to mention it... didn't want to create a new PR for it :)
